src/test/groovy/com/palantir/launchconfig/EclipseLaunchConfigTaskIntegrationSpec.groovy

```diff
@@ -149,6 +149,124 @@ class EclipseLaunchConfigTaskIntegrationSpec extends IntegrationSpec {
         fileExists("${projectName}-otherRun.launch")
     }
 
+    def "generates launch file using 'includedTasks' config"() {
+        setup:
+        writeHelloWorld("com.testing")
+
+        buildFile << """
+            apply plugin: 'java'
+            apply plugin: 'eclipse'
+            apply plugin: 'com.palantir.launch-config'
+
+            task runDev(type: JavaExec) {
+                classpath project.sourceSets.main.runtimeClasspath
+                main 'com.testing.HelloWorld'
+            }
+
+            task otherRun(type: JavaExec) {
+                classpath project.sourceSets.main.runtimeClasspath
+                main 'com.testing.HelloWorld'
+            }
+
+            launchConfig {
+				includedTasks = ['runDev']
```

Comment:
weird indent.

i would also prefer if this ended up being:

```groovy
launchConfig {
    includedTasks 'runDev', 'otherRun', 'etc'
}
```

also, thoughts on changing it to `include` and `exclude`?